AppController/djinn.rb

```diff
@@ -5656,4 +5675,54 @@ def stop_open()
     return
   end
 
+  # Gathers App Controller and System Manager stats for this node.
+  #
+  # Args:
+  #   secret: The secret of this deployment.
+  # Returns:
+  #   A hash in string format containing system and platform stats for this
+  #     node.
+  def get_monitors(secret)
```

Comment:
I think this name might be a little misleading since it returns info and not mechanisms for monitoring things.